_posts/newsletter/2021-03-11-this-month-in-solid.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: newsletter
+title: This Month in Solid 2020-03-11
+permalink: /newsletter/2020-03-11
+tags: [monthly, updates]
+categories: [Updates]
+---
+
+## Updates
+
+### [Events](https://solidproject.org/events)
+
+**11 March 2021** — It was exciting to see the great turnout at March’s Solid World. We continue to grow in the Solid community as reflected in Solid World’s highest attendance ever. During the event, we got to touch on a lot of great ideas related to the growing market of interoperability. 
```

Comment:
heh ... from the content of the newsletter, not sure if they cover interoperability.  But eh.
Also, "growing market of interoperability" - not sure what that phrase means, but eh again.

Reply:
@MarrelleBailey Want to change this, or shall we just publish as-is?

Reply:
Let's keep it. But I will keep a note of it